lib/teleterm/apiserver/handler/handler_unified_resources.go

```diff
@@ -52,6 +52,7 @@ func (s *Handler) ListUnifiedResources(ctx context.Context, req *api.ListUnified
 		UseSearchAsRoles:    req.GetSearchAsRoles(),
 		PinnedOnly:          req.GetPinnedOnly(),
 		IncludeRequestable:  req.GetIncludeRequestable(),
+		IncludeHealthStatus: true,
```

Comment:
i'm actually unsure about this.

this looks like a flag that should be controlled by the caller... i think we do use `ListResources` elsewhere that may not need the health status. does including health status slow things down?

what's everyone opinion?